### PR TITLE
Java client paging / sort api has wrong parameter

### PR DESCRIPTION
**Pull request by St0neWan9**
1 Change ‘count’ to ‘size’ to be in accordance with sensei json api

2 Change 'sorts' to 'sort', and change List<Sort> to Map<String, String> for encoding to json from 

"sorts": [{
"field": "time",
"order": "desc"
}]

to

"sort": [{
"time": "desc"
}]

### CODE REVIEW

File: clients/java/src/main/java/com/senseidb/search/client/json/req/SenseiClientRequest.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
    * initializing parameters that are needed by all runtime facet handlers
    */
   private Map<String, Map<String, FacetInit>> facetInit = new HashMap<String, Map<String, FacetInit>>();
-  private List<Sort> sorts = new ArrayList<Sort>();
```
Comment (alejandroperez) [inline clients/java/src/main/java/com/senseidb/search/client/json/req/SenseiClientRequest.java]: Why the change in the Sort type?

Comment (St0neWan9) [inline clients/java/src/main/java/com/senseidb/search/client/json/req/SenseiClientRequest.java]: If use the Sort List, encoded json string will like this 
"sorts": [{
"field": "time",
"order": "desc"
}]
which sensei can't handle it right.

Sensei takes this 
"sort": [{
"time": "desc"
}]
kind of json for sorting, so we change Sort List to Map<String, String> List like this:

``` java
private List<Map<String, String>> sort = new ArrayList<Map<String, String>>();
```

**Comment by St0neWan9** [comment]
Thanks Volodymyr's help, we've fixed the NPE issue.
